pkg/controller/issueapi/issue.go

```diff
@@ -143,8 +145,13 @@ func (c *Controller) HandleIssue() http.Handler {
 			}
 		}
 
+		var intAsBytes []byte
+		binary.LittleEndian.PutUint64(intAsBytes, uint64(id))
+		idString := base64.StdEncoding.EncodeToString(intAsBytes)
```

Comment:
You'll probably want URLEncoding here instead, since this will be used as part of a URL. However, it's probably better to use hex here (even though it's longer).

Reply:
Also document this 😄